frontend-mobile/lib/widgets/coffee_call.dart

```diff
@@ -61,39 +60,55 @@ class _CoffeeCallBodyState extends State<CoffeeCallBody> {
     if (userName == '') {
       return NoUsername(onUsernameSet: _setUsername);
     }
-    return Column(
-      children: [
-        Padding(
-          padding: const EdgeInsets.all(20),
-          child: Hero(
-            tag: 'coffee-cup',
-            child: Material(
-              child: ElevatedButton(
-                onPressed: _onCoffeeCupClicked,
-                style: ElevatedButton.styleFrom(
-                  shape: const CircleBorder(),
-                  padding: const EdgeInsets.all(48),
+    return BlocBuilder<CoffeeCallCubit, CoffeeCallState>(
+      builder: (context, state) {
+        return Container(
```

Comment:
Could be a padding (only use `Container`s if you need at least two of its properties)

Reply:
Done, thanks for the hint, makes sense :)